### PR TITLE
i1108: Update to not use deprecated methods

### PR DESCRIPTION
**Pull request by MartinEden**
https://vimc.myjetbrains.com/youtrack/issue/VIMC-1108

### CODE REVIEW

File: src/databaseInterface/src/main/java/org/vaccineimpact/api/db/Routines.java [hunk 1]
```diff
@@ -69,27 +68,6 @@ public static void enableTrigger(Configuration configuration, String tableName,
         p.execute(configuration);
     }
 
-    /**
```
Comment (littledice) [inline src/databaseInterface/src/main/java/org/vaccineimpact/api/db/Routines.java]: Isn't this file auto-generated? Similarly the one below?

**Comment by littledice** [comment]
Also TC is failing :(

File: src/databaseInterface/src/main/java/org/vaccineimpact/api/db/Routines.java (hunk 1 above)
Comment (MartinEden) [reply src/databaseInterface/src/main/java/org/vaccineimpact/api/db/Routines.java]: Yup. I've changed the xml file that configures how it is generated. See main-database.xml.

**Comment by MartinEden** [comment]
Fixed in TeamCity

File: src/build.gradle [hunk 2]
```diff
@@ -97,7 +97,12 @@ subprojects {
...
     }
 
-    tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+    compileKotlin {
+        kotlinOptions {
+            jvmTarget = "1.8"
+        }
+    }
+    compileTestKotlin {
         kotlinOptions {
```
Comment (MartinEden) [inline src/build.gradle]: This change isn't strictly necessary - it's just a refactor to improve readability.

**Pull request merged**